docs/guides/data-movement/readme.md

```diff
@@ -7,20 +7,18 @@ categories: provisioning
 
 Data Movement can be configured in multiple ways:
 
-1. Server side
+1. Server side (`NnfDataMovementProfile`)
 2. Per Copy Offload API Request arguments
 
-The first method is a "global" configuration - it affects all data movement operations. The second
-is done per the Copy Offload API, which allows for some configuration on a per-case basis, but is
-limited in scope. Both methods are meant to work in tandem.
+The first method is a "global" configuration - it affects all data movement operations that use a a
+particular `NnfDataMovementProfile` (or the default). The second is done per the Copy Offload API,
+which allows for some configuration on a per-case basis, but is limited in scope. Both methods are
+meant to work in tandem.
 
-## Server Side ConfigMap
+## `NnfDataMovementProfiles`
 
-The server side configuration is done via the `nnf-dm-config` config map:
-
-```bash
-kubectl -n nnf-dm-system get configmap nnf-dm-config
-```
+The server side configuration is done by creating `NnfDataMovementProfiles` resources in kubernetes.
+These work similar to `NnfStorageProfiles` #TODO LINK.
 
 The config map allows you to configure the following:
```

Comment:
Do you still have a config map?

Reply:
I think I popped some changed before I committed and didn't notice. I'll clean this up.